plugins/restful/RestfulDataProviderDbQuery.php

```diff
@@ -349,25 +349,43 @@ public function update($id, $full_replace = FALSE) {
     // Build the update array.
     $request = $this->getRequest();
     static::cleanRequest($request);
+    $save = FALSE;
+    $original_request = $request;
+
     $public_fields = $this->getPublicFields();
     $fields = array();
-    foreach ($public_fields as $public_property => $info) {
+    foreach ($public_fields as $public_field_name => $info) {
+      if (!empty($info['create_or_update_passthrough'])) {
+        // Allow passing the value in the request.
+        if (!empty($info['create_or_update_passthrough_required']) && !isset($request[$public_field_name])) {
```

Comment:
@mateu-aguilo-bosch @amitaibu The entity validator can handle schema validation and generic objects. You can look [here](https://github.com/Gizra/entity_validator/blob/7.x-1.x/modules/entity_validator_example/plugins/validator/entity_validator_example/EntityValidatorExampleObjectValidator.class.php)